app/controllers/candidate_interface/gcse/naric_reference_controller.rb

```diff
@@ -47,5 +47,16 @@ def naric_reference_params
       params.require(:candidate_interface_naric_reference_form)
         .permit(:naric_reference_choice, :naric_reference, :comparable_uk_qualification)
     end
+
+    def grade_edit_path
```

Comment:
This method appears in a number of places and could be extracted

Reply:
Some kind of `ResolveGcseEditPath` service object would be useful.